.github/workflows/ci-workflow.yml

```diff
@@ -355,11 +355,18 @@ jobs:
         env:
           ASAN_OPTIONS: "detect_leaks=1"
 
+      - name: Test fwd only rel tables with ASAN
```

Comment:
Good idea on testing here! I hope eventually we can cover all tests with fwd and bwd only storage.